README.md

```diff
@@ -135,27 +134,41 @@ config:
   environment:
     CME_MODEL:
       stage: before
-      value: "{CAPELLACOLLAB_SESSION_PROVISIONING[0][path]}"
+      value:
+        path: "{CAPELLACOLLAB_SESSION_PROVISIONING[0][path]}"
+        diagram_cache:
+          path: "{CAPELLACOLLAB_SESSION_PROVISIONING[0][diagram_cache]}"
+          password: "{CAPELLACOLLAB_SESSION_API_TOKEN}"
+          username: "{CAPELLACOLLAB_SESSION_REQUESTER_USERNAME}"
+        fallback_render_aird: "true"
+    CME_LOG_FILE: /var/log/session/model-explorer.log
+    CME_LIVE_MODE: "0"
     CME_ROUTE_PREFIX: "{CAPELLACOLLAB_SESSIONS_BASE_PATH}"
   connection:
     methods:
       - id: f51872a8-1a4f-4a4d-b4f4-b39cbd31a75b
         type: http
-        name: Direct Browser connection
-        sharing:
-          enabled: true
+        name: Browser connection
         ports:
           metrics: 8000
           http: 8000
+        sharing:
+          enabled: true
         redirect_url: "{CAPELLACOLLAB_SESSIONS_SCHEME}://{CAPELLACOLLAB_SESSIONS_HOST}:{CAPELLACOLLAB_SESSIONS_PORT}{CAPELLACOLLAB_SESSIONS_BASE_PATH}/"
   monitoring:
     prometheus:
       path: /metrics
+    logging:
+      enabled: true
   provisioning:
     directory: /models
     max_number_of_models: 1
+    required: false
+    provide_diagram_cache: true
```

Comment:
What does `provisioning.required: false` mean? We have it in our prod config, but I don't know when or why that was introduced there, and I also can't find it in the Collab Manager docs. (Or is this just the default?)

Reply:
Found docs: https://github.com/DSD-DBS/capella-collab-manager/blob/403345b096f00b47ec10157aff71e0f37933c59f/backend/capellacollab/tools/models.py#L324-L330

> Specifies if a tool requires provisioning. If enabled and a session without provisioning is requested, it will be declined.

This very much sounds like something we'd want for the model explorer. But at the same time I don't see any difference in how the frontend behaves - it won't let you start a session without selecting a model either way. (Haven't tested the API though.)